modules/tensor_mechanics/doc/content/modules/tensor_mechanics/NewBackground.md

```diff
@@ -0,0 +1,139 @@
+# Theory for the New Lagrangian Kernels
```

Comment:
Maybe rename this file to NewLagrangianKernels.md (to make it more clear what it's about from its name)?

Reply:
How about LagrangianKernelTheory.md

I made that rename change.

Reply:
Yes, I like that name.